content/en/docs/instrumentation/python/automatic/example.md

```diff
@@ -287,6 +307,47 @@ If those headers are available, they will be included in your span:
 }
 ```
 
+### Execute a programmatically instrumented server
```

Comment:
This section should be nested under `Execute` rather than `Configure`.

Reply:
Fixed